aten/src/ATen/core/boxing/impl/make_boxed_from_unboxed_functor.h

```diff
@@ -51,7 +51,7 @@ using supported_primitive_arg_types = guts::typelist::typelist<
         /* everything is ok, this is a primitive type */
       }, /* else */ [] {
         auto tmap = c10::getCustomClassTypeMap();
-        TORCH_CHECK(c10::isCustomClassRegistered<T>(), "Tried to use undefined class as input argument");
+        TORCH_CHECK(c10::isCustomClassRegistered<T>(), "Tried to use undefined class ", c10::util::get_fully_qualified_type_name<T>(), " as input argument");
```

Comment:
Don't know how you guys feel about this but I've been linewrapping this kind of stuff.